docs/outlook/onmessagesend-onappointmentsend-events.md

```diff
@@ -235,6 +235,78 @@ In Outlook on the web and in new Outlook on Windows (preview):
+>
+> To test this feature in Outlook on Windows, you must install Version 2403 (Build 17330.10000) or later. Then, join the [Microsoft 365 Insider program](https://insider.microsoft365.com/join/windows) and select the **Beta Channel** option in your Outlook client to access Office beta builds.
+
+You can use Markdown to format the message of a Smart Alerts dialog. However, only the following elements are supported.
+
+- Bold, italic, or bold and italic text. Both the [asterisk (*) and underscore (_) formats](https://www.markdownguide.org/basic-syntax/#emphasis) are supported.
+
+    ```javascript
+    event.completed({
+      allowEvent: false,
+      ...
+      errorMessageMarkdown: "**Important**: Apply the appropriate sensitivity label to your message before sending."
+    });
+    ```
+
+    :::image type="content" source="../images/outlook-smart-alerts-bold.png" alt-text="A sample Smart Alerts dialog with bold text.":::
+
+- Bulleted or unordered lists. To create an item in the list, begin with a dash (`-`) or asterisk (`*`), add the content, then append `\r` to signify item completion.
+
+    ```javascript
+    event.completed({
+      allowEvent: false,
+      ...
+      errorMessageMarkdown: "Your email doesn't meet company guidelines.\n\nFor additional assistance, contact the IT Service Desk:\n\n- Phone number: 425-555-0102\r- Email: it@contoso.com\r- Website: [Contoso IT Service Desk](https://www.contoso.com/it-service-desk)\r"
+    });
+    ```
+
+    :::image type="content" source="../images/outlook-smart-alerts-list.png" alt-text="A sample Smart Alerts dialog containing a bulleted list.":::
+
+- Numbered or ordered lists. To create an item in the list, begin with a number followed by a period, add the content, then append `\r` to signify item completion. The first item of the list must start with the number one (`1.`) and the succeeding numbers must be in numerical order.
+
+    ```javascript
+    event.completed({
+      allowEvent: false,
+      ...
+      errorMessageMarkdown: "Help your recipients know your intentions when you send a mail item. To set the sensitivity level of an item:\n\n1. Select **File** > **Properties**.\r2. From the **Sensitivity** dropdown, select **Normal**, **Personal**, **Private**, or **Confidential**.\r3. Select **Close**.\r"
+    });
+    ```
+
+    :::image type="content" source="../images/outlook-smart-alerts-numbered-list.png" alt-text="A sample Smart Alerts dialog containing a numbered list.":::
+
+- Links. To create a link, enclose your link text in square brackets (`[]`), then enclose the HTTPS URL in parentheses (`()`). The angle brackets format (`<>`) isn't supported.
+
```

Comment:
Should probably note that the link must be 'https' or it won't be interpreted as a link.

Reply:
Added.